lib/srv/usermgmt_linux.go

```diff
@@ -71,3 +75,31 @@ func (*HostUsersProvisioningBackend) DeleteUser(name string) error {
 	}
 	return trace.Wrap(err)
 }
+
+func (*HostUsersProvisioningBackend) TestSudoersFile(contents []byte) error {
+	code, err := host.TestSudoersFile(contents)
+	if err != nil {
+		if code != 0 {
+			return trace.Errorf("visudo: invalid sudoers file")
+		}
+		return trace.Wrap(err)
+	}
+	return nil
+}
+
+func (u *HostUsersProvisioningBackend) WriteSudoersFile(username string, contents []byte) error {
+	sudoersFilePath := filepath.Join(u.sudoersPath, fmt.Sprintf("%s-%s", "teleport", username))
```

Comment:
Is `username` guaranteed to not contain any illegal characters for a path?

Should we include the node's `host_uuid` in the path?